data/brands/amenity/atm.json

```diff
@@ -240,6 +240,70 @@
         "name:he": "בנק טינקוף",
         "name:ru": "Тинькофф Банк"
       }
+    },
+    {
+        "displayName": "高雄銀行",
+        "id": "bankofkaohsiung-9a6797",
+        "locationSet": {"include": ["tw"]},
+        "tags": {
+          "amenity":"atm",
+          "brand":"高雄銀行",
+          "brand:en":"Bank of Kaohsiung",
+          "brand:wikidata":"Q11673248",
+          "brand:zh":"高雄銀行",
+          "name":"高雄銀行",
+          "name:en":"Bank of Kaohsiung",
+          "name:nanKo":"hiông Gîn-hâng",
+          "name:nan:HJ": "高雄銀行",
+          "name:nan:POJ":"Ko-hiông Gîn-hâng",
+          "name:nan:TL":"Ko-hiông Gîn-hâng",
+          "name:zh":"高雄銀行",
+          "operator":"雄銀行",         
+          "operator:wikidata":"Q11673248",
+          "matchNames":"bok,高銀,高雄市銀行,高雄銀"
+        }
+      },
+      {
+          "displayName": "集友銀行 Chiyu Banking Corporation",
+          "id": "chiyubankingcorporation-e57d9a",
+          "locationSet": {"include": ["hk"]},
+          "tags": {
+            "amenity":"atm",
+            "brand":"集友銀行 Chiyu Banking Corporation",
+            "brand:en":"Chiyu Banking Corporation",
+            "brand:wikidata":"Q877121",
+            "brand:zh":"集友銀行",
+            "name":"集友銀行 Chiyu Banking Corporation",
+            "name:en":"Chiyu Banking Corporation",
+            "name:zh":"集友銀行",
+            "name:zh-Hans":"集友银行",
+            "name:zh-Hant":"集友銀行",
+            "operator":"集友銀行 Chiyu Banking Corporation",         
+            "operator:wikidata":"Q877121"
+          }
+        },
+        {
+            "displayName": "陽信商業銀行",
+            "id": "sunnycommercialbank-9a6797",
+            "locationSet": {"include": ["tw"]},
+            "tags": {
+              "amenity":"atm",
+              "brand":"陽信商業銀行",
+              "brand:en":"Sunny Commercial Bank",
+              "brand:wikidata":"Q11658759",
+              "brand:zh":"陽信商業銀行",
+              "name":"陽信商業銀行",
+              "name:en":"Sunny Commercial Bank",
+              "name:nan":"Iông-sìn Siong-gia̍p Gîn-hâng",
+              "name:nan:H0": "陽信商業銀行",
```

Comment:
It looks like no features in OSM are currently tagged with [`name:nan:H0`](https://taginfo.osm.org/keys/name:nan:h0)

Reply:
> It looks like no features in OSM are currently tagged with [`name:nan:H0`](https://taginfo.osm.org/keys/name:nan:h0)

This `name:nan:H0` maybe wrong typo of `name:nan:HJ` and should be changed to `name:nan-HJ`